packages/inventory-compliance/src/compliance.scss

```diff
@@ -63,6 +63,12 @@
     word-wrap: break-word;
 }
 
-.compliance-rules-table td[data-label=Severity] {
-    text-transform: capitalize;
+.compliance-rules-table {
+    td[data-label=Severity] {
+        text-transform: capitalize;
+    }
+
+    td[data-label=Rule] small {
+        color: #72767b; // var(--pf-c-content--small--Color)
```

Comment:
I have no clue why this variable isn't working. It seems like insights chrome is overriding the value to be `rgb(26, 26, 26)`

Reply:
I can't find any variable `--pf-c-content--small--Color` in PF core, perhaps you should use `var(--pf-global--Color--dark-200)` instead. Or you should wrap `Text` with `TextContent` component
```JSX
ruleTitleCell = (title, identifier) => <Stack>
        <StackItem>{ title }</StackItem>
        { identifier && <StackItem><TextContent><Text component={ TextVariants.small }>{ identifier.label }</Text></TextContent></StackItem> || '' }
    </Stack>
```

Reply:
Awesome, thanks!